modules/vaos/app/services/ccra/configuration.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Ccra
+  # CCRA::Configuration provides the configuration settings for the CCRA API.
+  # It retrieves settings from the application configuration (e.g., Settings.vaos.ccra)
+  # and delegates common REST parameters.
+  class Configuration < Common::Client::Configuration::REST
```

Comment:
Note: This class has some code that is duplicated in the EPS service and can be refactored. Will do that after an initial review to make sure I am on the right track.